src/overset/TiogaSTKIface.C

```diff
@@ -408,10 +408,20 @@ TiogaSTKIface::get_receptor_info()
   //   - the STK global ID for the donor element
   //
   size_t ncount = receptors.size();
-  for (size_t i=0; i<ncount; i+=3) {
+  stk::mesh::EntityId  donorID = std::numeric_limits<stk::mesh::EntityId>::max();
+#ifdef TIOGA_HAS_UINT64T
+  int rec_offset = 4;
```

Comment:
I'm not sure what rec_offset = 4 or 3 means. Can you put 4 and 3 in named variables?

Reply:
Thanks, I'll add comments that explains the meaning of 3 and 4.